validation/validate_foundation_year.py

```diff
@@ -0,0 +1,6 @@
+from django.core.exceptions import ValidationError
+
+
+def validate_foundation_year(foundation_year_value: int):
+    if len(str(foundation_year_value)) != 4:

```

Comment:
and what if foundation_year_value > current_year ?

Reply:
corrected, thank you!

Reply:
Done